rpc/rpc_machine.c

```diff
@@ -470,10 +470,13 @@ static int rpc_tm_setup(struct m0_net_transfer_mc *tm,
 	if (rc < 0)
 		return M0_ERR_INFO(rc, "TM initialization");
 
-	max_msgs_size = m0_rpc_max_msg_size(net_dom, msg_size);
+	if (net_dom->nd_xprt == &m0_net_lnet_xprt)
+		max_msgs_size = m0_rpc_max_msg_size(net_dom, msg_size);
+	else
+		max_msgs_size = 1;
```

Comment:
Here, we can just call:
```
max_msgs_size = m0_rpc_max_msg_size(net_dom, msg_size);
```
because the special case for lnet is already handled in m0_rpc_max_msg_size().

Right?